nrt/data/SMIB/RegulationProportional/SMIB_2_StepEfd/SMIB3.dyd

```diff
@@ -0,0 +1,44 @@
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, you can obtain one at http://mozilla.org/MPL/2.0/.
+    SPDX-License-Identifier: MPL-2.0
+
+    This file is part of Dynawo, an hybrid C++/Modelica open source time domain
+    simulation tool for power systems.
+-->
+<dyn:dynamicModelsArchitecture xmlns:dyn="http://www.rte-france.com/dynawo">
+  <!-- Network model -->
+  <dyn:modelicaModel id="Network">
+    <dyn:unitDynamicModel id="busInf" name="Dynawo.Electrical.Buses.InfiniteBus" parFile="SMIB.par" parId="1"/>
+    <dyn:unitDynamicModel id="line1" name="Dynawo.Electrical.Lines.Line" parFile="SMIB.par" parId="2"/>
+    <dyn:unitDynamicModel id="line2" name="Dynawo.Electrical.Lines.Line" parFile="SMIB.par" parId="3"/>
+    <dyn:unitDynamicModel id="tfo" name="Dynawo.Electrical.Transformers.TransformerFixedRatio" parFile="SMIB.par" parId="4"/>
+    <dyn:connect id1="busInf" var1="terminal" id2="line1" var2="terminal1"/>
+    <dyn:connect id1="busInf" var1="terminal" id2="line2" var2="terminal1"/>
+    <dyn:connect id1="line1" var1="terminal2" id2="tfo" var2="terminal1"/>
+    <dyn:connect id1="line2" var1="terminal2" id2="tfo" var2="terminal1"/>
+  </dyn:modelicaModel>
+  <!-- Generator with four windings with fixed excitation voltage and proportional speed regulation -->
+  <dyn:modelicaModel id="SM">
+    <dyn:unitDynamicModel id="gen" name="Dynawo.Electrical.Machines.GeneratorSynchronous" initName = "Dynawo.Electrical.Machines.GeneratorSynchronousExt_4E_INIT" parFile="SMIB.par" parId="5" />
+    <dyn:unitDynamicModel id="gover" name="Dynawo.Electrical.Controls.Machines.Governors.GoverProportional" initName="Dynawo.Electrical.Controls.Machines.Governors.Governor_INIT" parFile="SMIB.par" parId="6"/>
+    <dyn:unitDynamicModel id="Pm" name="Dynawo.Electrical.Controls.Basics.SetPoint" initName="Dynawo.Electrical.Controls.Basics.SetPoint_INIT" parFile="SMIB.par" parId="8"/>
+    <dyn:unitDynamicModel id="efd" name="Dynawo.Electrical.Controls.Basics.Step" initName="Dynawo.Electrical.Controls.Basics.Step_INIT" parFile="event.par" parId="1"/>
+    <dyn:unitDynamicModel id="omegaRef" name="Dynawo.Electrical.Controls.Basics.SetPoint" parFile="SMIB.par" parId="10"/>
+    <dyn:initConnect id1="gover" var1="Pm0Pu" id2="Pm" var2="Value0"/>
+    <dyn:initConnect id1="gen" var1="Pm0Pu" id2="gover" var2="Pm0Pu"/>
+    <dyn:initConnect id1="gen" var1="Efd0Pu" id2="efd" var2="Value0"/>
+    <dyn:connect id1="gover" var1="PmRefPu" id2="Pm" var2="setPoint"/>
+    <dyn:connect id1="gen" var1="PmPu" id2="gover" var2="PmPu"/>
+    <dyn:connect id1="gen" var1="omegaPu" id2="gover" var2="omegaPu"/>
+    <dyn:connect id1="gen" var1="efdPu" id2="efd" var2="step"/>
+    <dyn:connect id1="gen" var1="omegaRefPu" id2="omegaRef" var2="setPoint"/>
+  </dyn:modelicaModel>
+  <!-- Connections -->
+  <dyn:connect id1="Network" var1="tfo_terminal2" id2="SM" var2="gen_terminal"/>
+</dyn:dynamicModelsArchitecture>
```

Comment:
I haven't found a job using this dyd file at first glance

Reply:
true: deleted